### ISSUE TITLE
The performance seems to be bad with my datasets.

### ISSUE THREAD

**Issue by feifeiwei** [lucastabelini/PolyLaneNet#11]
hello, I have trained polyLanenet with my data, but the results performance are bad. 
the epochs are 250, the poly loss is around 0.1

**Comment by lucastabelini** [comment]
How many images does you dataset have? What settings did you use? There are many things that will affect the model's performance. Maybe if you provide more information I may be able to help you.

**Comment by feifeiwei** [comment]
There are about 35000 training images.
My setting are as follows: 
```
backbone: mobilenetv2
conf_weight: 1
  lower_weight: 1
  upper_weight: 1
  cls_weight: 0
  poly_weight: 300
  batchsize: 48
optimizer:
  name: Adam
  parameters:
    lr: 3.0e-4
img_size: h, w = [256, 512]
```

**Comment by lucastabelini** [comment]
You're using a backbone that I did not experiment with, so I can't say whether it should work or not. Which learning rate scheduler are you using? The one I used has a minimum at  `385 * (n + 1)` epochs. If you're using the same, in the epoch 250 it will not be at a minimum, and that can affect the model's performance, since the learning rate will still be quite high. Do you have any performance metrics on the test set? I can't really tell how bad it is based on 3 images.

Apart from that, I can't really help much without access to the code and data you're using.

**Comment by feifeiwei** [comment]
Thank your for your help, I would try another backbone, hope it will help.
